### DISCUSSION TITLE
I am getting this error please guide me for the same ,its showing me val images but not trained images

### DISCUSSION THREAD

**Discussion by Kedar0019** [ultralytics/ultralytics#8686] [Q&A]
AttributeError: 'FreeTypeFont' object has no attribute 'getsize'
                   all         13         15    0.00333      0.867    0.00958    0.00429
Speed: 5.7ms pre-process, 338.5ms inference, 0.0ms loss, 10.2ms post-process per image
Saving runs\detect\train7\predictions.json...
Results saved to runs\detect\train7

please help me for the same to resolve this error

**Comment by glenn-jocher** [comment]
@Kedar0019 hello! It looks like you're encountering an issue related to font handling in your environment, specifically with the `FreeTypeFont` object. This error might occur when trying to display or save images with annotated text (e.g., class labels, confidence scores) but the font size method `getsize()` is not accessible or recognized.

A quick solution to try is ensuring you have the latest versions of the relevant libraries, especially `Pillow`, as it handles image processing and font rendering in Python. You can update `Pillow` using pip:

```bash
pip install --upgrade Pillow
```

If the issue persists, you might want to check if the font file used for annotations is correctly specified and accessible in your script or configuration. Sometimes, explicitly setting the path to a valid font file can resolve such issues.

Feel free to share more details if the problem continues, and we'll be happy to help further! 😊
